x-pack/plugins/ml/public/application/explorer/explorer_charts/explorer_charts_container.js

```diff
@@ -167,18 +184,20 @@ function ExplorerChartContainer({
             </MlTooltipComponent>
           );
         }
-        return (
-          <MlTooltipComponent>
-            {(tooltipService) => (
-              <ExplorerChartSingleMetric
-                tooManyBuckets={tooManyBuckets}
-                seriesConfig={series}
-                severity={severity}
-                tooltipService={tooltipService}
-              />
-            )}
-          </MlTooltipComponent>
-        );
+        if (chartType === CHART_TYPE.SINGLE_METRIC) {
```

Comment:
Probably one for a follow up, but would be nice if the `SINGLE_METRIC` and `DISTRIBUTION` charts expanded to the height of the map in this situation (preferable than shrinking the map height to match those chart heights I think).

![image](https://user-images.githubusercontent.com/7405507/106125553-36d4e080-6154-11eb-8cf7-356744bcac84.png)